--- a/vendor/typo3/cms-workspaces/Classes/Service/GridDataService.php
+++ b/vendor/typo3/cms-workspaces/Classes/Service/GridDataService.php
@@ -184,10 +184,12 @@
                 $versionArray['icon_Workspace'] = $iconWorkspace->getIdentifier();
                 $versionArray['icon_Workspace_Overlay'] = $iconWorkspace->getOverlayIcon()?->getIdentifier() ?? '';
                 $languageValue = $this->getLanguageValue($table, $versionRecord);
+                $languages = $this->getSystemLanguageValue($languageValue, $pageId, 'title') ? $this->getSystemLanguageValue($languageValue, $pageId, 'title') : 'all';
+
                 $versionArray['language'] = [
                     'icon' => $this->iconFactory->getIcon($this->getSystemLanguageValue($languageValue, $pageId, 'flagIcon') ?? 'empty-empty', IconSize::SMALL)->getIdentifier(),
-                    'title' => $this->getSystemLanguageValue($languageValue, $pageId, 'title'),
-                    'title_crop' => htmlspecialchars(GeneralUtility::fixed_lgd_cs($this->getSystemLanguageValue($languageValue, $pageId, 'title'), (int)$backendUser->uc['titleLen'])),
+                    'title' => $languages,
+                    'title_crop' => htmlspecialchars(GeneralUtility::fixed_lgd_cs($languages, (int)$backendUser->uc['titleLen'])),
                 ];
                 if ($isAllowedToPublish && $swapStage === StagesService::STAGE_PUBLISH_ID && (int)$versionRecord['t3ver_stage'] === StagesService::STAGE_PUBLISH_ID) {
                     $versionArray['allowedAction_publish'] = $isRecordTypeAllowedToModify && $this->stagesService->getStage($stages, StagesService::STAGE_PUBLISH_ID)->isAllowed;
